spec/spec.md

```diff
@@ -186,7 +186,7 @@ The following process defines how a DID-Relative URL for a Decentralized Web Nod
 **DID-Relative URL example for passing multiple messages:**
 
 ::: note
-For example purposes, the `message` parameter value below is neither JSON stringified nor Base64Url encoded, but should be when using Decentralized Web Node URLs in practice (see the [DID-relative URL Composition](#composition) instructions above).
+***Note:** For example purposes, the `message` parameter value below is neither JSON stringified nor Base64Url encoded, but should be when using Decentralized Web Node URLs in practice (see the [DID-relative URL Composition](#composition) instructions above).*
 :::
```

Comment:
```suggestion
```

Reply:
This already renders as a note in UI, so there's no real need to again say Note within the note.